packages/snap/package.json

```diff
@@ -7,37 +7,26 @@
     "url": "https://github.com/rarimo/rarime.git"
   },
   "license": "(MIT-0 OR Apache-2.0)",
-  "files": [
```

Comment:
files and `.npmignore` conflicts with each other. Better to use npm ignore if we want to granularly exclude files

Reply:
They don't conflict. They complement each other. With `"files"` you whitelist assets and folders you want to publish, with `.npmignore` you exclude the assets that are in the whitelist. Using just `.npmignore` doesn't scale because you'll need to add every new file there